bulk/sensor_bulk_load-AssetRecord.csv

```diff
@@ -1,2568 +1,2573 @@
-CGINS-FLORTM-02760,N00301,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2760,,20121002,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02761,N00306,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2761,,20121002,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02762,N00311,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2762,,20121206,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02786,N00316,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2786,,20121206,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02806,N00321,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2806,,20121228,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02807,N00326,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2807,,20121228,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02809,N00729,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2809,,20130125,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02812,N00132,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2812,,20130301,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02813,N00032,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2813,,20130412,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02817,N00147,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2817,,20130701,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02818,N00142,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2818,,20130701,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02819,N00118,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2819,,20130701,186935,CGSN Glider Instrument Pool,
-CGINS-FLORTM-02858,N00016,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,2858,,20130125,,CGSN Glider Instrument Pool,
-CGINS-FLORTM-03015,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SLK,3015,,,,CGSN Glider Instrument Pool,Woods Hole Oceanographic Institution
-CGINS-FLORTM-03027,,Sensor,1,3-Wavelength Fluorometer for Glider: FLORT Series M,FLORT for Coastal Slocum Glider,WET Labs,ECO Puck FLBBCD-SL
```

Comment:
I had already reviewed the sensor additions; I think the whole file is now showing as changed because of Stuart changing the line endings to be Unix. So I think this is ok.